db/migrate/20160214115800_inline_ems_id.rb

```diff
@@ -28,7 +28,7 @@ def add_columns
 
   def update_columns
     say_with_time("Inline ems_id in containers") do
-      Container.includes(:container_definitions => :container_group).all.each do |container|
+      Container.includes(:container_definition => :container_group).all.each do |container|
```

Comment:
I'd like a deeper review on this one before merge.

@moolitayer I don't understand how this ever worked as the [relation is singular](https://github.com/ManageIQ/manageiq/pull/15069/files#diff-0591b4a672ef7861abe16191fab2ebc9R11).  The only thing I can think of is that on [line 26](https://github.com/ManageIQ/manageiq/pull/15069/files#diff-0591b4a672ef7861abe16191fab2ebc9R26) a new column called container_definitions was added, and somehow that was used instead of the relation?  I really don't understand this fix.

When I run this migration in a standalone dummy Rails app I get the following error, unless I fix it as done here.

```
  1) InlineEmsId#up it sets the value of container.ems_id to the value of container.container_group.ems_id
     Failure/Error: described_class.migrate(migration_direction)

     ActiveRecord::AssociationNotFoundError:
       Association named 'container_definitions' was not found on InlineEmsId::Container; perhaps you misspelled it?
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations.rb:238:in `association'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:161:in `block in grouped_records'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:159:in `each'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:159:in `grouped_records'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:148:in `preloaders_for_one'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:125:in `block in preloaders_for_hash'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:124:in `each'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:124:in `flat_map'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:124:in `preloaders_for_hash'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:113:in `preloaders_on'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:102:in `block in preload'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:101:in `each'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:101:in `flat_map'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/associations/preloader.rb:101:in `preload'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/relation.rb:708:in `block in exec_queries'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/relation.rb:707:in `each'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/relation.rb:707:in `exec_queries'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/relation.rb:583:in `load'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/relation.rb:260:in `records'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/relation/delegation.rb:38:in `each'
     # ./db/migrate/20160214115800_inline_ems_id.rb:31:in `block in update_columns'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:815:in `block in say_with_time'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:815:in `say_with_time'
     # ./db/migrate/20160214115800_inline_ems_id.rb:30:in `update_columns'
     # ./db/migrate/20160214115800_inline_ems_id.rb:21:in `up'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:792:in `exec_migration'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:773:in `block (2 levels) in migrate'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:772:in `block in migrate'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/connection_adapters/abstract/connection_pool.rb:398:in `with_connection'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:771:in `migrate'
     # /Users/jfrey/.gem/ruby/2.4.1/gems/activerecord-5.0.2/lib/active_record/migration.rb:601:in `migrate'
     # ./spec/support/migration_helper.rb:71:in `migrate_under_test'
     # ./spec/support/migration_helper.rb:35:in `block (2 levels) in migrate'
     # ./spec/support/migration_helper.rb:83:in `suppress_migration_messages'
     # ./spec/support/migration_helper.rb:35:in `block in migrate'
     # ./spec/support/migration_helper.rb:57:in `clearing_caches'
     # ./spec/support/migration_helper.rb:31:in `migrate'
     # ./spec/migrations/20160214115800_inline_ems_id_spec.rb:16:in `block (3 levels) in <top (required)>'
     # ./spec/support/migration_helper.rb:57:in `clearing_caches'
     # ./spec/support/migration_helper.rb:13:in `block (2 levels) in migration_context'
```